target/linux/lantiq/xrx200/config-5.4

```diff
@@ -17,6 +17,7 @@ CONFIG_CRYPTO_ZSTD=y
 CONFIG_EXTRA_FIRMWARE="lantiq/xrx200_phy11g_a14.bin lantiq/xrx200_phy11g_a22.bin lantiq/xrx200_phy22f_a14.bin lantiq/xrx200_phy22f_a22.bin"
 CONFIG_EXTRA_FIRMWARE_DIR="firmware"
 CONFIG_GENERIC_ALLOCATOR=y
+CONFIG_GRO_CELLS=y
```

Comment:
Is this line necessary?

Reply:
not for an upstream kernel
but it's needed because of an downstream patch in OpenWrt: ``target/linux/generic/backport-5.4/755-v5.8-net-dsa-add-GRO-support-via-gro_cells.patch``